### DISCUSSION TITLE
gns3 2.2.27  gns3_controller is empty

### DISCUSSION THREAD

**Discussion by jonfon96** [GNS3/gns3-server#2000] [General]
I installed Gns3 2.2.27 vm 2.2.27 vmware player 16.2.1 , on Dell Laptop 32 gig Ram , 500 GIg HD 
I have had some issue with this install version I am not able to keep the IOU and ios router and switch as well as the Palo alto FW.  

when I installed all the app came up without issue I enable GNs3 VM and it stated the VM all see ok but when I shutdown the app and restateded it was as if I had a new install.   I repeated the process again and same results.  I was told to look at the gns3_controller file and see if I can enable the setting but that shows up as empty file.  I would like to know if I need to down this version to 2.2.25 or .24 since I would like to test out the newest options. 

thank you

**Comment by josephmhiggins** [comment]
the original post is here in the gns3 community:
https://gns3.com/community/featured/gns3-2-2-27-2
